low-level/files/opt/seagate/sspl/setup/generate_sspl_bundle/sspl_bundle_generate

```diff
@@ -319,26 +293,22 @@ def initialize_logging(parser):
         _logger.addHandler(console)
     return _logger
 
+logger = initialize_logging(sys.argv)
+
 if __name__ == "__main__":
-    if not os.path.isfile(sspl_sb_req):
-        if len(sys.argv) >= 3:
-            # Initialize logging
-            logger = initialize_logging(sys.argv)
-            tmpFile = open(sspl_sb_req, "w")
-            os.makedirs(sspl_bundle_tmp)
-            try:
-                SSPLBundleObj = SSPLBundleGenerate()
-                SSPLBundleObj.generate_sspl_support_bundle(sys.argv, tmpFile)
-            except Exception as e:
-                logger.error("SSPL support bundle script is failed with an unexpected "
-                    "error. please try again ..!")
-                logger.error("Error Message : %s" % e)
-                raise SSPLBundleError(1, msg)
-            tmpFile.close()
+    if len(sys.argv) >= 3:
+        try:
+            SSPLBundleObj = SSPLBundleGenerate()
+            SSPLBundleObj.generate_sspl_support_bundle(sys.argv)
+        except Exception as e:
+            logger.error("SSPL support bundle script is failed with an unexpected "
+                "error. please try again ..!")
+            msg = "Error Message : %s" % e
+            logger.error(msg)
             sspl_bundle_cleanup()
-        else:
-            msg = "Unrecognized arguments: %s" % sys.argv
             raise SSPLBundleError(1, msg)
+        sspl_bundle_cleanup()
     else:
-        msg = "Already SSPL Support Bundle request is going on, So skipping current request..!"
-        raise Exception(msg)
+        msg = "Unrecognized arguments: %s" % sys.argv
+        sspl_bundle_cleanup()
```

Comment:
why clean up on wrong trigger? If someone triggers sb reqesut with invalid number of arguments and lets say there was one ongoing request, this cleanup will end up interfering in that request also, like cleaning those tmp files

Reply:
Right, Removed